hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/TokenFeature.java

```diff
@@ -1093,6 +1131,107 @@ private List<Long> getSerialsForToken(TokenId tokenId) {
                 : nftInfoForToken.stream().map(NftInfo::serialNumber).toList();
     }
 
+    private NetworkTransactionResponse airdropFungibleTokens(
+            TokenId tokenId, int amount, ExpandedAccountId sender, ExpandedAccountId receiver) {
+
+        networkTransactionResponse =
+                tokenClient.executeFungibleTokenAirdrop(tokenId, sender, receiver.getAccountId(), amount);
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
+
+        return networkTransactionResponse;
+    }
+
+    private NetworkTransactionResponse cancelTokenAirdrops(
+            ExpandedAccountId sender, AccountId receiver, TokenId tokenId) {
+        networkTransactionResponse = tokenClient.executeCancelTokenAirdrop(sender, receiver, tokenId);
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
+
+        return networkTransactionResponse;
+    }
+
+    private TokenAirdrop getPendingAirdrop(TokenId tokenId, AccountId sender, AccountId receiver) {
+        return mirrorClient.getPendingAirdrops(receiver).getAirdrops().stream()
+                .filter(tokenAirdrop -> tokenAirdrop.getTokenId().equals(tokenId.toString())
+                        && tokenAirdrop.getSenderId().equals(sender.toString())
+                        && tokenAirdrop.getReceiverId().equals(receiver.toString()))
+                .findFirst()
+                .orElse(null);
+    }
+
+    private TokenAirdrop getOutstandingAirdrop(TokenId tokenId, AccountId sender, AccountId receiver) {
+        return mirrorClient.getOutstandingAirdrops(sender).getAirdrops().stream()
+                .filter(tokenAirdrop -> tokenAirdrop.getTokenId().equals(tokenId.toString())
+                        && tokenAirdrop.getSenderId().equals(sender.toString())
+                        && tokenAirdrop.getReceiverId().equals(receiver.toString()))
+                .findFirst()
+                .orElse(null);
+    }
+
+    private NetworkTransactionResponse claimTokenAirdrops(
+            ExpandedAccountId sender, ExpandedAccountId receiver, TokenId tokenId) {
+        networkTransactionResponse = tokenClient.executeClaimTokenAirdrop(sender, receiver, tokenId);
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
+
+        return networkTransactionResponse;
+    }
+
+    private void verifyTokenAirdrop(TokenAirdrop tokenAirdrop, AccountId sender, AccountId receiver, TokenId tokenId) {
+        assertThat(tokenAirdrop).isNotNull();
+        assertThat(tokenAirdrop.getReceiverId()).isEqualTo(receiver.toString());
+        assertThat(tokenAirdrop.getSenderId()).isEqualTo(sender.toString());
+        assertThat(tokenAirdrop.getTokenId()).isEqualTo(tokenId.toString());
```

Comment:
Always chain when you can.

```suggestion
        assertThat(tokenAirdrop)
            .isNotNull()
            .returns(receiver.toString(), TokenAirdrop::getReceiverId)
            .returns(sender.toString(), TokenAirdrop::getSenderId)
            .returns(tokenId.toString(), TokenAirdrop::getTokenId);
```

Reply:
done